src/ng-accessibility-bar.component.js

```diff
@@ -93,8 +105,14 @@
             }
 
             if (isInFontSizeLimit) {
-                operator === '+' ? defaultFontSize += vm.config.fontAdjust.incrementRate : defaultFontSize -= vm.config.fontAdjust.incrementRate;
+                if (operator === '+') {
+                    defaultFontSize += vm.config.fontAdjust.incrementRate;
+                } else {
+                    defaultFontSize -= vm.config.fontAdjust.incrementRate;
+                }
+
                 $html.setAttribute('style', 'font-size: ' + defaultFontSize + 'px;');
+                _switchClass('html', currentFontClassName, prefixFontClassName + defaultFontSize);
```

Comment:
Fala aí ! Ótima ideia ter essa classe auxiliar para quando for alterado o tamanho da fonte.
Única coisa que acho que poderíamos alterar aqui é adicionar a classe ao body como fazemos no contraste.
O que você acha ?